contents/javascript/Javascript-framework-Angular.md

```diff
@@ -30,12 +30,22 @@ Consider a simple web application that displays the name of a person as shown be
     The sandbox above is editable. You can enter your name in the text box and see how the UI is automatically updated. You can also view and edit the code by clicking on the Both button. 
 </box>
 
-The UI of an Angular app is defined in _templates_ which contain a mix of HTML code and HTML-like custom tags.
+The UI of an Angular app is defined in _templates_ which contain a mix of HTML code and HTML-like custom tags. The template for the above example is given below.
+
+``` html
+<h1>About Me!</h1>
+<h2>Hello World! My name is {hero.name}</h2>
+<div>
+<label>NAME:
+  <input [(ngModel)]="hero.name"/>
+</label>
+</div>
+```
   
```

Comment:
Would this also work? It's closer to the layout in the page.
```html
<label>NAME: <input [(ngModel)]="hero.name"/></label>
```

Reply:
Yes, this will also work. Will change this